### ISSUE TITLE
[BUG] HUD stopped showing after upgrading to Foudry version 12 for D&D4.

### ISSUE THREAD

**Issue by Leeder87** [Larkinabout/fvtt-token-action-hud-core#268]
After upgrading my Foundry and corresponding D&D4 module up to 12 version, the HUD stopped working. I tried different browsers, but none is showing.

**Versions:**
Token Action HUD Core: 1.5.7
Token Action HUD Classic: 3.0.21
System: D&D 4th Edition version 0.5.4
Foundry VTT: Version 12 build 331
Browsers: latest Firefox, Edge, Chrome

**Comment by Larkinabout** [comment]
I don't think there's an existing module to support the D&D 4e system in v12 yet. Someone in the 4e community would need to create one.

**Comment by draconas1** [comment]
TAH support is part of Drac's 4e tools and has not had a TAH 2.0 upgrade yet. 

Hopefully will by the end of the weekend, I'm just updating now.

**Comment by Larkinabout** [comment]
> TAH support is part of Drac's 4e tools and has not had a TAH 2.0 upgrade yet.
Ah, that'll be why I couldn't find anything for it in Foundry's pages. I think @Leeder87 may have been using Token Action HUD Classic though previously, so they might want to move over to using Drac's module.

**Comment by draconas1** [comment]
Yeah, I put it in there at it was easier at the time and meant i wasn't maintaining more repos, though given now the hard break between 1.5 and 2 I should bite the bullet and release it as a separate module.

**Comment by draconas1** [comment]
Forked out.  Please Use https://github.com/draconas1/token-action-hud-dnd4e   That is TAH 2.0.x compatible.  

You will also want to upgrade to the latest versions of Drac's Tools.
